packages/webpack-plugin/src/plugin.ts

```diff
@@ -404,7 +419,11 @@ export class StylableWebpackPlugin {
                          * They might be used by other stylesheets so they might end up in the final build
                          */
                         for (const request of stylableBuildMeta.unusedImports) {
-                            module.addDependency(new this.entities.UnusedDependency(request));
+                            module.addDependency(
+                                new this.entities.UnusedDependency(
+                                    this.stylable.resolver.resolvePath(module.context!, request)
```

Comment:
I think its not the place to resolve this. a better place would be where we create `unusedImports`. I will also need to double check this change to make sure we don't somehow use the request field on `UnusedDependency`.

Reply:
this is changed in #2786